### PR TITLE
import knative/test-infra scripts in here

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

The goal is to not depend on `knative` anymore and tailor these scripts to our needs.
/cc @bobcatfish wdyt ?

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by dlorenc** [comment]
> The goal is to not depend on `knative` anymore and tailor these scripts to our needs.
> /cc @bobcatfish wdyt ?

The goal makes sense to me, and this seems like a reasonable starting point!

**Comment by bobcatfish** [comment]
i think in the long run we don't want to have these scripts at all, and and to replace them with `Tasks` that have similar functionality - if you think that importing the knative/test-infra scripts will get us closer to that then im all for it :D

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
+# Helper scripts
+
+This directory contains helper scripts used by Prow test jobs, as well and
```
Comment (dlorenc) [inline scripts/README.md]: nit: typo. I think this should be "as well as"

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+   in the repo.
+
+1. [optional] Define the functions `pre_unit_tests()` and/or
+   `post_unit_tests()`. These functions will be called before or after the
+   unit tests (either your custom one or the default action) and will cause
+   the test to fail if they don't return success.
+
+1. [optional] Define the function `integration_tests()`. If you don't define
+   this function, the default action for running the integration tests is to run
+   all run all `./test/e2e-*tests.sh` scripts, in sequence.
```
Comment (dlorenc) [inline scripts/README.md]: nit: run all twice.

**Comment by vdemeester** [comment]
@dlorenc fixed

**Comment by bobcatfish** [comment]
amazing README @vdemeester :sob: :bowing_woman: !!!

**Pull request merged**